API.md

```diff
@@ -56,6 +57,11 @@
         - [Updating user](#updating-user)
         - [Getting user](#getting-user)
         - [Deleting user](#deleting-user)
+    - User Tokens
+        - [Listing tokens](#listing-tokens)
```

Comment:
This link and the three following after this one are currently broken because the headings are named differently. Based on their names it should be:

+ `Listing user tokens` => `#listing-user-tokens`
+ `Creating a user token` => `#creating-a-user-token`
+ `Updating a user token` => `#updating-a-user-token`
+ `Deleting a user token` => `#deleting-a-user-token`